app/scripts/metamask-controller.js

```diff
@@ -2117,29 +2117,19 @@ export default class MetamaskController extends EventEmitter {
       },
     );
 
-    let rpcUrlOrigin;
-    try {
-      rpcUrlOrigin = new URL(rpcUrl).origin;
-    } catch {
-      // ignore
-    }
     this.metaMetricsController.trackEvent({
       event: 'Custom Network Added',
       category: EVENT.CATEGORIES.NETWORK,
       referrer: {
-        url: rpcUrlOrigin,
+        url: ORIGIN_METAMASK,
       },
       properties: {
         chain_id: chainId,
         network_name: chainName,
-        network: rpcUrlOrigin,
```

Comment:
Could `network_name` and/or `block_explorer_url` be considered for removal as well? Don't seem super when `chain_id` is there and increase cardinality (anyone with custom unique enough RPC name and/or self-hosted explorer will be singled out)

Reply:
Just to give an opposite opinion for consideration: 
- By knowing `network_name` we can notice that [200 people have added BSC Testnet as Ethereum Classic](https://mixpanel.com/s/3djnfJ). Is it just by mistake or some rising scam going on? Knowing that could help us investigate and act upon it. We don't have anyone actively looking at it at the moment though so it's ok if we prefer removing it. 
- Agree with removing `block_explorer_url` . It could be interesting to replace it though by a boolean that just tells us wether or not a block explorer has been included (as it's not mandatory to include one). Some parts of the product assume the user has a valid block explorer added and they will not work if the user hasn't (this is specially true for mobile where storage constraints are significant and a lot of the user ability to review their activity relies on us redirecting them to the block explorer). Knowing the % that has a block explorer added versus not could help us make product decisions to rely less or more on block explorers.

Reply:
Re the `network_name` point: This could similarly be re-added in a safe way (say, by emitting only predefined networks with enough usage, and have a common value for all `other`), which could cover for scenarios like the mentioned one.

Reply:
Done.